<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#GameWindow{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1001</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="buildPage">
    <property name="styleSheet">
     <string notr="true">QWidget#buildPage{
	background-image: url(:/img/img/apocalypseBackgroundGameWindow.jpg);
}</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>631</width>
       <height>591</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="buildGrid"/>
    </widget>
    <widget class="QPushButton" name="finishButton">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>520</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#finishButton{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(225, 225, 225);
	font: 25pt &quot;Algerian&quot;;
	color: black;
}

QWidget#finishButton::pressed{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(182, 182, 182);
	font: 25pt &quot;Algerian&quot;;
	color: black;
}
</string>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backToMainMenuButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#backToMainMenuButton{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(225, 225, 225);
	font: 15pt &quot;Algerian&quot;;
	color: black;
}

QWidget#backToMainMenuButton::pressed{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(182, 182, 182);
	font: 15pt &quot;Algerian&quot;;
	color: black;
}</string>
     </property>
     <property name="text">
      <string>Back to Main Menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="selectBlocksOuterLabel">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>70</y>
       <width>311</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#selectBlocksOuterLabel{
	background-color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>80</y>
       <width>293</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="buildBlocksVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="placeBlocksLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#placeBlocksLabel{
	border-width: 3px;
	border-style: solid;
	border-color: rgb(0, 255, 0);
	background-color: white;
	font: 9pt &quot;8514oem&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Place Blocks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="budgetLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#budgetLabel{
	border-width: 3px;
	border-style: solid;
	border-color: rgb(255, 0, 0);
	background-color: white;
	font: 9pt &quot;8514oem&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>(Show Budget)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="grassLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#grassLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Grass ($2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="grassImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="grassCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="woodLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#woodLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Wood ($3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="woodImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="woodCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="cobblestoneLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#cobblestoneLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Cobblestone ($5)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cobblestoneImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cobblestoneCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="concreteLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#concreteLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Concrete ($7)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="concreteImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="concreteCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="selectDisasterOuterLabel">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>300</y>
       <width>311</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#selectDisasterOuterLabel{
	background-color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>310</y>
       <width>291</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="disasterVerticalLayout">
      <item>
       <widget class="QLabel" name="selectDisasterLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#selectDisasterLabel{
	border-width: 3px;
	border-style: solid;
	border-color: rgb(0, 0, 255);
	background-color: white;
	font: 9pt &quot;8514oem&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
        </property>
        <property name="text">
         <string>Select Disaster</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="floodLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#floodLabel{
	background-color: white;
	font: 600 15pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Flood</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="floodImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="floodCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="wildfireLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#wildfireLabel{
	background-color: white;
	font: 600 14pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Wildfire</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="wildfireImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wildfireCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="tornadoLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#tornadoLabel{
	background-color: white;
	font: 600 14pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>Tornado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tornadoImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tornadoCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>520</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#resetButton{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(225, 225, 225);
	font: 25pt &quot;Algerian&quot;;
	color: black;
}

QWidget#resetButton::pressed{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(182, 182, 182);
	font: 25pt &quot;Algerian&quot;;
	color: black;
}
</string>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="simulationPage">
    <property name="styleSheet">
     <string notr="true">QWidget#simulationPage{
	background-image: url(:/img/img/simulationPageBackground.jpg);
}</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>711</width>
       <height>621</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="simulationGrid"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>290</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="entityInformationVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="playerLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#playerLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Player</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playerImageLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="robberLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#robberLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Robber</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="robberImageLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="terroristLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#terroristLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Terrorist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="terroristImageLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="disasterLabel">
          <property name="styleSheet">
           <string notr="true">QLabel#disasterLabel{
	background-color: white;
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
          </property>
          <property name="text">
           <string>Disaster</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="disasterImageLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>490</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="survivalTimeVerticalLayout">
      <item>
       <widget class="QLabel" name="survivalTimeDisplayLabel">
        <property name="styleSheet">
         <string notr="true">QLabel#survivalTimeDisplayLabel{
	background-color: white;
	font: 600 14pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
        </property>
        <property name="text">
         <string>Survival Time (seconds)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="survivalTimeLabel">
        <property name="styleSheet">
         <string notr="true">QLabel#survivalTimeLabel{
	background-color: white;
	font: 600 14pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="upArrowKeyLabel">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>40</y>
       <width>61</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="leftArrowKeyLabel">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>90</y>
       <width>62</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="downArrowKeyLabel">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>140</y>
       <width>61</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rightArrowKeyLabel">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>90</y>
       <width>62</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="useArrowKeysToMoveLabel">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>210</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#useArrowKeysToMoveLabel{
	background-color: white;
	font: 600 11pt &quot;Segoe UI Semibold&quot;;
	qproperty-alignment: AlignCenter;
}</string>
     </property>
     <property name="text">
      <string>Use Arrow Keys to Move</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameOverPage">
    <property name="styleSheet">
     <string notr="true">QWidget#gameOverPage{
	background-image: url(:/img/img/gameOverPageBackground.jpg);
}</string>
    </property>
    <widget class="QLabel" name="causeOfDeathLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>981</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#causeOfDeathLabel{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.735577 rgba(255, 255, 0, 255), stop:0.927885 rgba(255, 0, 0, 255), stop:0.995192 rgba(0, 0, 0, 255));
	font: 26pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="gameOverSurvivalTimeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>911</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#gameOverSurvivalTimeLabel{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));
	font: 9pt &quot;8514oem&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="gameOverPageBackToMainMenuButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#gameOverPageBackToMainMenuButton{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(225, 225, 225);
	font: 15pt &quot;Algerian&quot;;
	color: black;
}

QWidget#gameOverPageBackToMainMenuButton::pressed{
	border-width: 4px;
	border-style: solid;
	border-color: black;
	background-color: rgb(182, 182, 182);
	font: 15pt &quot;Algerian&quot;;
	color: black;
}</string>
     </property>
     <property name="text">
      <string>Back to Main Menu</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="gameOverDisasterTypeHorizontalLayout">
      <item>
       <widget class="QLabel" name="gameOverDisasterLabel">
        <property name="styleSheet">
         <string notr="true">QLabel#gameOverDisasterLabel{
	background-color: white;
	font: 9pt &quot;8514oem&quot;;
	qproperty-alignment: AlignCenter;
	color: black;
}</string>
        </property>
        <property name="text">
         <string>Disaster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameOverDisasterImageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
